HISTORY.rst

```diff
@@ -30,7 +30,7 @@ v3.3.5 (2023 Jan 5)
 - Add "Setting Up Your Environment with the Python API" to Python Docs (python-api docs).
 - [Python API Documentation] Update Python version requirements.
 - Rename Shotgun to Shotgrid in every about text like tk-multi-demo git repository.
-- Rename Shotgun servers to ShotGrid servers in the documentation.
+- Rename Shotgun servers to Flow Production Tracking servers in the documentation.
```

Comment:
I don't think we should change these ones. The title of the document, yes. But we are not changing the history.